src/unix/syscalls.ml

```diff
@@ -0,0 +1,13 @@
+external pread : Unix.file_descr -> int64 -> bytes -> int -> int -> int
+  = "caml_pread"
+
+let pread ~fd ~fd_offset ~buffer ~buffer_offset ~length =
+  pread fd fd_offset buffer buffer_offset length
+
+external pwrite : Unix.file_descr -> int64 -> bytes -> int -> int -> int
+  = "caml_pwrite"
+
+let pwrite ~fd ~fd_offset ~buffer ~buffer_offset ~length =
+  pwrite fd fd_offset buffer buffer_offset length
+
+external fsync : Unix.file_descr -> unit = "unix_fsync"
```

Comment:
as we are exporting these symbols a bit more publicly now, maybe it's a good time to check that have consistent names. Why `caml_pread` but `unix_fsync`? :-)

Reply:
we should make sure they won't clash with symbols defined in `tezos-stdlib-unix` and `extunix`

Reply:
`tezos-stdlib-unix` doesn't have any external definitions. `extunix` has the following:
- `caml_extunix_fsync`
- `caml_extunix_pread64`
- `caml_extunix_pwrite64`

I have no particular thoughts on what sort of naming scheme we should use. Anyone want to give an opinion?

Reply:
use `caml_index_{op}` seems the safest bet.

Reply:
Done.